src/main/webapp/onelive/archive/index.html

```diff
@@ -154,6 +77,54 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
 <script>
     $(function () {
         loadNavAndFooter('/assets/content/static');  //relative path to content directory
+        loadYoutubeVideos();
     });
+
+    //function to load youtube videos
+    function loadYoutubeVideos() {
+        $.ajax({
+            type: 'get',
+            url: 'https://sef-dataholder.herokuapp.com/youtube/onelive',
+            dataType: 'json',
+            success: function (data) {
+                //mustache render
+                let renderedData = Mustache.render(
+                    $('#template-youtube-vidoe').html(), {data: data});
+                $('#youtube-video').html(renderedData);
+            }
+        });
+    }
+</script>
+
+<script type="text/html"
+        id="template-youtube-vidoe">
+    <div class="row justify-content-center">
+        <div class="col-9 text-center">
+            <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+            <p class="display-3 text-onelive font-weight-bolder">One Lives</p>
+        </div>
+    </div>
```

Comment:
You can put static parts like this in the body itself. Let's keep only the dynamic parts on the template.

Reply:
Ok